exporter.go

```diff
@@ -159,14 +104,14 @@ func (b *Exporter) handleEvent(event Event) {
 			errorEventStats.WithLabelValues("empty_metric_name").Inc()
 			return
 		}
-		metricName = escapeMetricName(mapping.Name)
```

Comment:
is there anything that prevents us from making the mapper return an escaped name in the first place?